css-view-transitions-2/Overview.bs

```diff
@@ -598,20 +615,19 @@ When capturing the old or new state for an element, perform the following steps
 			Note: this means that calling {{Document/startViewTransition()}} while capturing the old document for a cross-document view-transition would run the callback but skip the animation.
 
 		1. Let |outboundTransition| be a new {{ViewTransition}} object in |oldDocument|'s [=relevant Realm=],
-			whose [=ViewTransition/active types=] is |resolvedRule|, and whose [=ViewTransition/process old state captured=] is set to the following steps:
+			whose [=ViewTransition/active types=] is |resolvedRule|.
 
-			Issue: should we check for the opt-in again, in case there was a CSSOM change in a requestAnimationFrame callback?
+		1. Run the following steps when [=Perform pending transition operations|performing pending transition operations=]:
 
-			1. If |outboundTransition|'s [=ViewTransition/phase=] is "`done`", then call |onReady| and return.
+			1. If |outboundTransition|'s [=ViewTransition/phase=] is "`done`", then call |proceedWithNavigation| and return.
 
 			1. Assert: |outboundTransition|'s  [=ViewTransition/phase=] is "`pending-capture`".
 
-			1. [=Clear view transition=] |outboundTransition|.
+			1. Set |oldDocument|'s [=auto-skip view transitions=] to false.
 
-				Note: The ViewTransition object on the old Document should be destroyed after its state has been copied to the new Document below.
-					We explicitly clear it here since the old Document may be cached by the UA.
+			1. [=Capture the old state=] for |outboundTransition|.
```

Comment:
We need to handle the fact that capture is async, see note [here](https://drafts.csswg.org/css-view-transitions-1/#setup-view-transition:~:text=A%20task%20is%20queued%20here%20because%20the%20texture%20read%20back%20in%20capturing%20the%20image%20may%20be%20async%2C%20although%20the%20render%20steps%20in%20the%20HTML%20spec%20act%20as%20if%20it%E2%80%99s%20synchronous.).

I think that means setting the rendering suppression bit to true here and unsetting both rendering suppression + auto-skip VT in the task queued below?

Reply:
Reminder for this. We need to set rendering suppression for view transitions to true here if capture old state succeeds and post a task to run the success steps which also reset this bit. This is to allow async capture.